pm/sender.go

```diff
@@ -155,12 +154,17 @@ func (s *sender) validateTicketParams(ticketParams *TicketParams, numTickets int
 		return fmt.Errorf("ticket faceValue %v > max faceValue %v", ticketParams.FaceValue, maxFaceValue)
 	}
 
+	latestBlock := s.timeManager.LastSeenBlock()
+	if ticketParams.ExpirationBlock == nil || ticketParams.ExpirationBlock.Cmp(latestBlock) <= 0 {
```

Comment:
Actually `ticketParams.ExpirationBlock` can never be nil here now that I think of it. Because when converting `net.TicketParams` to `pm.TicketParams` we essentially set it to be `new(big.Int).SetBytes(nil)` which would be `0` and not `nil` 

Therefore I think what we would need to do to support the features you describe here is to add a check before this block 

```
if ticketParams.ExpirationBlock.Int64() == 0 { return nil }
```

Reply:
fixed in 0dbcf97df79a4ff27e77f4cc545ff7e9a11950cb